common/cmake/devices.cmake

```diff
@@ -1470,7 +1470,7 @@ function(ADD_FPGA_TARGET)
     append_file_dependency(VPR_DEPS ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     get_file_location(SDC_LOCATION ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     set(SDC_ARG --sdc_file ${SDC_LOCATION})
-    set(SDC_FILE ${SDC_LOCATION})
```

Comment:
This will not work generally speaking.  If the SDC is generated, this will break.

Reply:
This is in the if branch where the SDC is manually written. The case in which the SDC is auto-generated, the actual generated SDC is taken instead: https://github.com/SymbiFlow/symbiflow-arch-defs/blob/7183cb5e59d50955686c12e4b800a55fafd1fae5/common/cmake/devices.cmake#L1461-L1467

Reply:
Even in that cases, the user can provide a generated SDC, e.g.:
```
add_file_target(FILE my_generated.sdc GENERATED)
add_fpga_target(
 ...
 SDC_FILE my_generated.sdc
 )
```

The new code will break in this case.

Reply:
I have changed approach and now the symlinks are resolved before packaging results into the tarball